### PR TITLE
Add Chinese HK and Chinese TW translation

### PR DESCRIPTION
**Pull request by zxx-457**

### CODE REVIEW

**Comment by yanfali** [comment]
Hi. Can you add a read me about what the script and the data file are for? And where did it come from? I'm reluctant to add a 300k file unless I know where it's from. Thanks

**Comment by zxx-457** [comment]
The large file just words mapping

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "howler": "^2.1.3",
     "keypress.js": "^2.1.5",
     "lodash": "4.17.21",
+    "simplify-traditional": "^1.0.0",
```
Comment (yanfali) [inline package.json:34]: move this to dev dep please. it's not required for runtime.

**Comment by yanfali** [comment]
is there anyway to pull in the translations file dynamically from the original repo as a dep?

File: package.json (hunk 1 above)
Comment (zxx-457) [reply package.json:34]: Sure , I,m sorry

I will write a readme

**Comment by zxx-457** [comment]
> is there anyway to pull in the translations file dynamically from the original repo as a dep?

Yes it could pull the dict from the repo , and dynamic translate , but I think this Kanji map doesn't change much.

**Comment by yanfali** [comment]
> > is there anyway to pull in the translations file dynamically from the original repo as a dep?
> 
> Yes it could pull the dict from the repo , and dynamic translate.
> 
> I will update the translate repo later for dynamic, thank you

Thank you. I just don't want to bloat up the repo with a file which is only used infrequently. Thanks!

**Comment by zxx-457** [comment]
> > > is there anyway to pull in the translations file dynamically from the original repo as a dep?
> > 
> > 
> > Yes it could pull the dict from the repo , and dynamic translate.
> > I will update the translate repo later for dynamic, thank you
> 
> Thank you. I just don't want to bloat up the repo with a file which is only used infrequently. Thanks!

Ok

**Comment by zxx-457** [comment]
Should I just add this two translates, and remove the script?

**Comment by yanfali** [comment]
> Should I just add this two translates, and remove the script?

Do you think we should run the script as part of the build process?

**Comment by zxx-457** [comment]
> > Should I just add this two translates, and remove the script?
> 
> Do you think we should run the script as part of the build process?

No, only use it when add simplified Chinese translates

**Comment by yanfali** [comment]
> > > Should I just add this two translates, and remove the script?
> > 
> > 
> > Do you think we should run the script as part of the build process?
> 
> No, only use it when add simplified Chinese translates

I'm ok adding the script to the project. I'll let you choose. Thank you for this contribution!

**Comment by zxx-457** [comment]
> > > > Should I just add this two translates, and remove the script?
> > > 
> > > 
> > > Do you think we should run the script as part of the build process?
> > 
> > 
> > No, only use it when add simplified Chinese translates
> 
> I'm ok adding the script to the project. I'll let you choose. Thank you for this contribution!

I think we could add it .

Thank you sir , I would like to contribute more If I'm capable.

**Comment by yanfali** [comment]
OK. I'll check it out locally tomorrow, and if it looks good I'll merge it.

**Comment by yanfali** [comment]
@noroadsleft please take a look

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Review by yanfali — APPROVED** [review]
Thank you.

**Pull request merged**